workflows_api/runtime/src/schemas.py

```diff
@@ -181,6 +182,9 @@ def object_is_accessible(cls, values):
         )
         return values
 
+    class Config:
+        extra = Extra.allow
+
```

Comment:
this shouldn't need extras

Reply:
This one is for `S3Input` and if we need to allow extra attributes inside discovery_items, we need this as well.

Reply:
we don't need to allow extra attributes inside `discovery_items`